### ISSUE TITLE
Question about 3D reconstruction.

### ISSUE THREAD

**Issue by imnaren142** [YadiraF/PRNet#94]
Hi,can we use this code for reconstruction the full head of the human being?

**Comment by AviP1234** [comment]
No, this code is trained on a database based on the Basel model that contains only the face. If you can get 3d head scan database you may re-train the code.

**Comment by fashionguy** [comment]
@naren142 Hi, I also want to get reconstruction of full head, including face and back. Do you find any good project? Can you share, please.
